sample_config/application.rb

```diff
@@ -62,39 +62,42 @@ class Application < Decko::Application
     config.autoload_paths += Dir["#{root}/test/*.rb"]
     config.autoload_paths += Dir["#{root}/test/**/"]
 
-    # config.file_buckets = {
-    #     s3_live_bucket: {
-    #         provider: "fog/aws",
-    #         directory: "wikirate",
-    #         subdirectory: "files",
-    #         attributes: { "Cache-Control" => "max-age=#{365.day.to_i}" },
-    #         public: true,
-    #         credentials: {
-    #           provider: 'AWS', # required
-    #           aws_access_key_id: 'XXXXXXX', # required
-    #           aws_secret_access_key: 'XXXXXX', # required
-    #           region: 'eu-central-1', # optional, defaults to 'us-east-1'
-    #         },
-    #         read_only: true,
-    #         # if public is set to false, following option is needed:
-    #         authenticated_url_expiration: 180
-    #     },
-    #     s3_staging_bucket: {
-    #         provider: "fog/aws",
-    #         directory: "wikirate",
-    #         subdirectory: "files",
-    #         attributes: { "Cache-Control" => "max-age=#{365.day.to_i}" },
-    #         public: true,
-    #         credentials: {
-    #           provider: 'AWS', # required
-    #           aws_access_key_id: 'XXXXXXX', # required
-    #           aws_secret_access_key: 'XXXXXX', # required
-    #           region: 'eu-central-1', # optional, defaults to 'us-east-1'
-    #         },
-    #         read_only: true,
-    #         # if public is set to false, following option is needed:
-    #         authenticated_url_expiration: 180
-    #     }
-    # }
+    config.file_buckets = {
+        s3_live_bucket: {
+            provider: "fog/aws",
+            directory: "wikirate",
+            subdirectory: "files",
+            credentials: {
+                provider: 'AWS', # required
+                aws_access_key_id: 'AKIAI2D7KEXYCST5XUBQ', # required
+                aws_secret_access_key: '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', # required
```

Comment:
This is a public repo. You don't want to commit any private information (like secret access keys) to a public repo.